examples/lustre/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# ch-test-scope: full
+# ch-test-arch-exclude: aarch64
+# No lustre RPMS for aarch64
+FROM centos7
+
+# install development tools
+RUN yum install -y epel-release
+
+# more dependencies
+RUN yum -y install libyaml-devel \
+                   wget \
+                   gcc
+
+ENV LUSTRE_URL=https://downloads.whamcloud.com/public/lustre/latest-release/el7/client/RPMS/x86_64
```

Comment:
Is it worth a `LUSTRE_VERSION` too?

Reply:
If we wanted to use a static version of Lustre yes. As we are only using the userspace tools I expect there won't be a problem with using the latest release. But that is dangerous thinking, as it very well could come back to bite us. We could settle on a version and update it sometimes as needed. If we do that we might want a system to make sure it doesn't get too stale.

Reply:
Behold as if by magic not having a lustre version has impacted me. I'll fix that.